CONTRIBUTING.md

```diff
@@ -190,7 +190,7 @@ For Docker compose `up` or `build` commands, you can use our Apple M1 override f
 
 # for development
 $ docker compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.apple-m1.override.yml up
-# only once: init admin user and create indexes and contraints in Neo4j database
+# only once: init admin user and create indices and constraints in Neo4j database
```

Comment:
Indexes is the correct namimg, see headline of:
https://neo4j.com/docs/cypher-manual/current/indexes-for-search-performance/

```suggestion
# only once: init admin user and create indexes and constraints in Neo4j database
```